code/game/objects/effects/decals/cleanable/humans.dm

```diff
@@ -120,6 +129,35 @@
 	bloodiness = 0
 	already_rotting = TRUE
 
+/obj/effect/decal/cleanable/blood/no_rot_gibs/up
+	icon_state = "gibup1"
+	random_icon_states = list("gib1", "gib2", "gib3", "gib4", "gib5", "gib6","gibup1","gibup1","gibup1")
+
+/obj/effect/decal/cleanable/blood/no_rot_gibs/down
+	icon_state = "gibdown1"
+	random_icon_states = list("gib1", "gib2", "gib3", "gib4", "gib5", "gib6","gibdown1","gibdown1","gibdown1")
+
+/obj/effect/decal/cleanable/blood/no_rot_gibs/body
+	icon_state = "gibtorso"
+	random_icon_states = list("gibhead", "gibtorso")
+
+/obj/effect/decal/cleanable/blood/no_rot_gibs/torso
+	icon_state = "gibtorso"
+	random_icon_states = null
+
+/obj/effect/decal/cleanable/blood/no_rot_gibs/limb
+	icon_state = "gibleg"
+	random_icon_states = list("gibleg", "gibarm")
+
+/obj/effect/decal/cleanable/blood/no_rot_gibs/core
+	icon_state = "gibmid1"
+	random_icon_states = list("gibmid1", "gibmid2", "gibmid3")
+
+/obj/effect/decal/cleanable/blood/no_rot_gibs/old
+	name = "old rotting gibs"
+	desc = "Space Jesus, why didn't anyone clean this up? They smell terrible."
+	bloodiness = 0
```

Comment:
This should be a false, I hope? Please god don't tell me this is a like, incremental tracker of bloodiness.

Reply:
It's a 0-100 value that has to do with the alpha of the bloody footprints. From cleanable.dm